doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -58,50 +58,68 @@ for backporting fixes to their stable kernel: check with your vendor.
 
 Synopsis
 ========
-In general, the command to mount CephFS via kernel driver looks like this::
+This is the general form of the command for mounting CephFS via the kernel driver:
 
-  mount -t ceph {device-string}={path-to-mounted} {mount-point} -o {key-value-args} {other-args}
+.. prompt:: bash #
+
+   mount -t ceph {device-string}={path-to-mounted} {mount-point} -o {key-value-args} {other-args}
 
 Mounting CephFS
 ===============
-On Ceph clusters, CephX is enabled by default. Use ``mount`` command to
-mount CephFS with the kernel driver::
+CephX is enabled by default in Ceph clusters. Use the ``mount`` command to
```

Comment:
CephX authentication 

perhaps.  I’ve never been sure of the proper capitalization.

Reply:
Accepted. We'll leave the matter of capitalization for future litigation.